### PR TITLE
support pyclipper

### PR DESCRIPTION
**Pull request by haoran1062**
### Description
I compiled the [pyclipper](https://github.com/fonttools/pyclipper) to pyodide, and successed with my computer. I also write test for it, I'm happy to do something for that great project, thanks.

### Checklists

add changelog

### CODE REVIEW

File: packages/pyclipper/test_pyclipper.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from pyodide_test_runner import run_in_pyodide
+
+
+@run_in_pyodide(packages=["pyclipper"])
+def test_mytestname(selenium):
```
Comment (ryanking13) [inline packages/pyclipper/test_pyclipper.py:5]: Please update the test name

**Review by ryanking13 — APPROVED** [review]
Thanks, @haoran1062! LGTM, please update the changelog.

File: packages/pyclipper/test_pyclipper.py (hunk 1 above)
Comment (haoran1062) [reply packages/pyclipper/test_pyclipper.py:5]: ok, I'll update the test name.

**Comment by haoran1062** [comment]
> Thanks, @haoran1062! LGTM, please update the changelog.

got it, updated.

**Comment by ryanking13** [comment]
Thanks a lot @haoran1062!

**Pull request merged**